ert_storage/graphql/ensembles.py

```diff
@@ -33,10 +41,43 @@ def resolve_parent_ensemble(
             return update.ensemble_reference
         return None
 
+    def resolve_unique_responses(
+        root: ds.Ensemble, info: "ResolveInfo"
+    ) -> Iterable[ds.Record]:
+        session = info.context["session"]
```

Comment:
You'll get a mypy error here because graphene's types are broken.

```suggestion
        session = info.context["session"]  # type: ignore
```